### PR TITLE
Making references to ClientHello and ServerHello consistent.

### PR DESCRIPTION
**Pull request by seanturner**

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -1384,16 +1384,16 @@ user_canceled
   by a close_notify.  This message is generally a warning.
 
 no_renegotiation
-: Sent by the client in response to a hello request or by the server
-  in response to a client hello after initial handshaking. Versions
+  Sent by the client in response to a ServerHello request or by the server
```
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: s/ServerHello request/ServerHello/

**Comment by seanturner** [comment]
Sorry 'bout all that me re-learning git again.

**Comment by davegarrett** [comment]
Assuming I've learned git properly, fixing the above should just be:
git reset --hard HEAD~2
git push -f
(done on this branch, of course; check branch before pushing)

**Comment by martinthomson** [comment]
Whenever you do a `git push -f`, always, ALWAYS specify a destination.  You can be screwed by your config if you don't.

```
git push -f origin chshnaming
```

The alternative is to do this:

```
git checkout HEAD~2
git checkout -b chshnamingv2
```
and open a new pull request.  No force pushes that way (just a dead branch somewhere).

**Pull request merged**